### ISSUE TITLE
Handle lack of `app/templates/head.hbs` more gracefully.

### ISSUE THREAD

**Issue by rwjblue** [ronco/ember-cli-head#7]
Currently, if you don't have the `app/templates/head.hbs` file present you get a very difficult to parse error like the following:

```
Uncaught Error: Could not find module `web-client/templates/head` imported from `web-client/components/head-content
```

It is pretty hard to track down unless you happen to know what ember-cli-head is doing internally.  I think erroring hard is ok here, but it is also possible for us to default to an empty template (to prevent errors like this).

**Comment by ronco** [comment]
@rwjblue (Sorry for the late reply.)  Is the blueprint that creates one on install insufficient?  Was it manually removed?  Seems pretty straight forward to add in, just curious how it came up.

**Comment by openhouse** [comment]
@ronco I got this error when migrating to pods.  I moved `web-client/templates/head.hbs` in a batch with all my other files and it ended up in `web-client/pods/head/template.hbs`.  I saw the error, moved it back and now it's the only file in my templates directory.

I know 'pods are dead' as we wait for the new module layout. But I thought using pods for the ui files was a good incremental step in my project.

```
loader.js:212Uncaught Error: Could not find module `web-client/templates/head` imported from `web-client/components/head-content`
```
